docs/schema/workflowconfig-element.md

```diff
@@ -43,19 +44,12 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Child elements
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="template-element-workflowconfig.md">Template Element (WorkflowConfig)</a></span></p>
-<p><span sdata="link"><a href="association-element-workflowconfig.md">Association Element (WorkflowConfig)</a></span></p>
-<p><span sdata="link"><a href="contenttypes-element-workflowconfig.md">ContentTypes Element (WorkflowConfig)</a></span></p>
-<p><span sdata="link"><a href="initiation-element-workflowconfig.md">Initiation Element (WorkflowConfig)</a></span></p></td>
-</tr>
-</tbody>
-</table>
+|Child Elements|
+|:-|
+|[Template Element (WorkflowConfig)](template-element-workflowconfig.md)|
```

Comment:
Please use 3 dashes between the pipes.  Less than 3 can cause issues.

Line 48: |:---|